bin/main.rb

```diff
@@ -1,2 +1,42 @@
 #!/usr/bin/env ruby
-puts 'Helo World!!!!'
+
+# Welcome users and show the board
+
+puts 'Welcome to the tic-tac-toe game'
+
+# Below is where the board will be displayed
+
+puts '| || || |'
+puts '-----------'
+puts '| || || |'
+puts '-----------'
+puts '| || || |'
+puts 'The winner occupies any horizontal, vertical or diagonal positions concurrently'
+
+# Tell players the rules of the game and what is expected of them
+
+puts 'Letters, spaces and special characters are not allowed!! Choose a number from 1 to 9'
+
+# Display the current positions that are taken, at the begining it will be empty.
+puts "Player One it's your turn"
```

Comment:
- [ ] `[REQUIRED]` game asks the player to select from available moves.
- [ ] `[REQUIRED]` game informs the player if the selected move is invalid.
- [ ] `[REQUIRED]` game displays board after the player move.
- [ ] `[REQUIRED]` game informs the player if the selected move is a winning move.
- [ ] `[REQUIRED]` game informs the player if the selected move is a draw move.
- [ ] `[REQUIRED]` game repeats all actions for the next player's move.